script/test

```diff
@@ -123,17 +107,11 @@ main()
     BOARDS=()
     if [ "${all_boards}" = true ]; then
         BOARDS+=(
-            "${MG1_BOARDS[@]}"
-            "${MG12_BOARDS[@]}"
-            "${MG13_BOARDS[@]}"
             "${MG21_BOARDS[@]}"
             "${MG24_BOARDS[@]}"
```

Comment:
We should probably add an MG26 board (brd4116a) to this list.

Reply:
- [ ] add brd4116a